gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/04_add_portal_menu_links_table.yml

```diff
@@ -10,6 +10,7 @@ databaseChangeLog:
               - column: {name: environment_id, type: nvarchar(64), constraints: { nullable: false } }
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.